process/factory/shard/vmContainerFactory.go

```diff
@@ -1,11 +1,13 @@
 package shard
 
 import (
-	"github.com/ElrondNetwork/arwen-wasm-vm/arwen"
-	arwenHost "github.com/ElrondNetwork/arwen-wasm-vm/arwen/host"
-	ipcCommon "github.com/ElrondNetwork/arwen-wasm-vm/ipc/common"
-	ipcMarshaling "github.com/ElrondNetwork/arwen-wasm-vm/ipc/marshaling"
-	ipcNodePart "github.com/ElrondNetwork/arwen-wasm-vm/ipc/nodepart"
+	arwen1_2 "github.com/ElrondNetwork/arwen-wasm-vm/arwen"
+	arwenHost1_2 "github.com/ElrondNetwork/arwen-wasm-vm/arwen/host"
+	ipcCommon1_2 "github.com/ElrondNetwork/arwen-wasm-vm/ipc/common"
```

Comment:
no more need for IPC - please remove.

Reply:
If we remove the IPC, we should also remove the flag from config that allowed to switch between the OOP and IPC

Reply:
IPC is needed because backwards compatibility.